code/_onclick/observer_onclick.dm

```diff
@@ -93,6 +93,8 @@
 		user.forceMove(get_turf(target))
 
 /obj/machinery/atmospherics/attack_ghost(mob/dead/observer/user)
+	if(!istype(user)) // Make sure user is actually an observer. Revenents also use attack_ghost, but do not have the toggle gas analyzer var.
```

Comment:
```suggestion
	if(!isobserver(user)) // Make sure user is actually an observer. Revenents also use attack_ghost, but do not have the toggle gas analyzer var.
```
Would using isobserver be better for this case?

Reply:
no

Reply:
Disregard. I see the issue now.